OutOfSchool/OutOfSchool.EmailSender/SendGridAccessibilityService.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Logging;
+
+using Quartz;
+
+namespace OutOfSchool.EmailSender;
+
+public interface ISendGridAccessibilityService 
+{
+    bool IsSendGridAccessible { get; set; }
+}
+public class SendGridAccessibilityService : ISendGridAccessibilityService
+{
+    public bool IsSendGridAccessible { get; set; }
```

Comment:
(Logic) I doubt if only 1 `bool` property would be enough. I'm expecting for some date of deprecation and related logic.

Reply:
Closing this one, but issue seems to be not resolved. See my separate (bigger) comment below